boot/zephyr/main.c

```diff
@@ -326,6 +330,9 @@ void main(void)
     BOOT_LOG_INF("Starting bootloader");
 
     os_heap_init();
+#if defined(CONFIG_SOC_SERIES_NRF53X) && defined(CONFIG_DEBUG)
+    enable_network_core_debug_pins();
```

Comment:
We could just leave `CONFIG_ENABLE_NETCORE` enabled and it will include similar functionality. However we would then start MCUBoot with a state where the network core is already online.

This means we would have to change `boot/zephyr/nrf53_cpunet_ctl.c:21 to first turn it off then turn it on.